run_all.py

```diff
@@ -25,11 +25,20 @@ def get_parser(mandatory=None):
         help='Path to config file, which contains parameters for the command sct_run_batch.',
     )
     parser.add_argument(
-        '-o-shell',
-        help='Path to the temporary batch script given to sbatch. By default basename is given a suffix _i with i the '
-             'number of iterations over the batch script and extension .sh. Example: -o-shell job_csa_sublist will '
-             'output job_csa_sublist_i.sh',
-        default='job_csa_sublist'
+        '-job-template',
+        help="""Path to sbatch config file containing sbatch options preceded of #SBATCH. Example: 
+             #SBATCH --account=def-jcohen
+             #SBATCH --time=0-08:00        # time (DD-HH:MM)
+             #SBATCH --nodes=1
+             #SBATCH --cpus-per-task=32    # number of OpenMP processes
+             #SBATCH --mem=128G
+             cd $SCRATCH""",
+    )
+    parser.add_argument(
+        '-n',
+        help="Break down OpenMP jobs across sub-datasets of n subjects. Adjust 'n' based on the number of CPU cores "
+             "available",
+        default=32
```

Comment:
```suggestion
        type=int,
        default=32
```

to avoid the following bug:
~~~
(venv-py36) [jcohen@gra-login1 ~]$ run_all -config config-csa-atrophy-t1.yml -job-template job-csa.sh -n 32
Traceback (most recent call last):
  File "/home/jcohen/csa-atrophy/venv-py36/bin/run_all", line 33, in <module>
    sys.exit(load_entry_point('csa-atrophy', 'console_scripts', 'run_all')())
  File "/home/jcohen/csa-atrophy/run_all.py", line 89, in main
    sublists = [list_subjects[i:i + n] for i in range(0, len(list_subjects), n)]
TypeError: 'str' object cannot be interpreted as an integer
~~~